testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerWellKnownProviderTest.java

```diff
@@ -158,10 +167,13 @@ public void testMinimalJwtCredentialHardcodedTest() {
         Assert.assertEquals(clientScope.getName(), supportedConfig.getCredentialDefinition().getType().get(0));
         Assert.assertEquals(1, supportedConfig.getCredentialDefinition().getContext().size());
         Assert.assertEquals(clientScope.getName(), supportedConfig.getCredentialDefinition().getContext().get(0));
-        Assert.assertNull(supportedConfig.getDisplay());
+        Assert.assertNotNull(supportedConfig.getCredentialMetadata());
         Assert.assertEquals(clientScope.getName(), supportedConfig.getScope());
 
-        compareClaims(supportedConfig.getFormat(), supportedConfig.getClaims(), clientScope.getProtocolMappers());
+        // Claims are now under credential_metadata, so we need to check if credential_metadata exists
+        if (supportedConfig.getCredentialMetadata() != null) {
+            compareClaims(supportedConfig.getFormat(), supportedConfig.getCredentialMetadata().getClaims(), clientScope.getProtocolMappers());
+        }
```

Comment:
You just asserted that it is not null.
```suggestion
            compareClaims(supportedConfig.getFormat(), supportedConfig.getCredentialMetadata().getClaims(), clientScope.getProtocolMappers());
```